### ISSUE TITLE
User account/profile metadata fields not being saved

### ISSUE THREAD

**Issue by 19belmont** [toddsundsted/ktistec#100]
Hi there,

In v2.0.0-10, when trying to add metadata fields data to the user’s profile, they are not saved.

Silent fail in the UI, I’m trying to see how to inspect any logged errors within the docker container, but I’m quite the n00b to all this, sorry.

**Comment by 19belmont** [comment]
Oh, it just worked. How odd.

I’ll try adding more.

**Comment by 19belmont** [comment]
Yeah, another added OK and showing from Mastodon website when viewing the profile.

I hate issues like this :/

**Comment by toddsundsted** [comment]
in my case it ended up being user error. there are two update buttons on the page. the upper one submit account changes (which includes the metadata), the bottom submits changes to the site settings. when i thought i could reproduce this it was because i was changing a value and then zooming to the bottom of the page to hit submit. the ux could probably use improvement.

**Comment by 19belmont** [comment]
I suspect I did the same :) I know when I pressed update today and it worked, I was making sure I was pressing the upper one.

I guess this can be closed or left open as a UX change request?
